### ISSUE TITLE
Linux client hangs at 'Configuring'

### ISSUE THREAD

**Issue by FrankWeis** [eduvpn/python-eduvpn-client#505]
# Installation

From repository, as per doc provided

# Version

3.1.0_1

# OS/Distribution

Ubuntu 22.04

# Logs
2022-09-09 16:19:39,092 - oauth-refresh - INFO - eduvpn.remote - remote.py:25 - Requesting https://eduvpn.mysite.xx/.well-known/vpn-user-portal
2022-09-09 16:19:39,170 - load-server-info - INFO - eduvpn.remote - remote.py:25 - Requesting https://eduvpn.mysite.xx/.well-known/vpn-user-portal
2022-09-09 16:19:39,236 - configure-connection - INFO - eduvpn.remote - remote.py:25 - Requesting https://eduvpn.mysite.xx/.well-known/vpn-user-portal
2022-09-09 16:19:40,499 - MainThread - INFO - eduvpn.nm - nm.py:294 - writing wireguard configuration to Network Manager
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/eduvpn/interface/event.py", line 204, in connect
    connection.connect(connected_callback)
  File "/usr/lib/python3/dist-packages/eduvpn/connection.py", line 100, in connect
    nm.start_wireguard_connection(
  File "/usr/lib/python3/dist-packages/eduvpn/nm.py", line 308, in start_wireguard_connection
    for dns_entry in config['Interface']['DNS'].split(','):
  File "/usr/lib/python3.10/configparser.py", line 1258, in __getitem__
    raise KeyError(key)
KeyError: 'DNS'

## Do you have a problem during connecting?

As soon as I set defaultGateway to false, the client hangs indefinately at 'Configuring' after I 'Approve' the application in the web interface. The Android client is not bothered by this setting. 
This happens also when I remove the dnsServerList.
This is a wireguard-only server.
Thanks

**Comment by jwijenbergh** [comment]
Thanks for reporting and the detailed information. Can you test #506?

Steps to test:

```bash
# Clone the branch
git clone https://github.com/jwijenbergh/python-eduvpn-client -b fix-dns-optional-wg

# Go to directory
cd fix-dns-optional-wg

# Run the client
python3 -m eduvpn.ui
```

**Comment by jwijenbergh** [comment]
Thanks for testing @FrankWeis. I have merged the PR so that the fix is in master. Keep in mind that a new release will take some time as there is a big refactor/rewrite of the app in progress currently

**Comment by ghudgens** [comment]
> Thanks for testing @FrankWeis. I have merged the PR so that the fix is in master. Keep in mind that a new release will take some time as there is a big refactor/rewrite of the app in progress currently

Are there no plans for any future bug fix/security updates for 3.1.X?

**Comment by jwijenbergh** [comment]
> > Thanks for testing @FrankWeis. I have merged the PR so that the fix is in master. Keep in mind that a new release will take some time as there is a big refactor/rewrite of the app in progress currently
> 
> Are there no plans for any future bug fix/security updates for 3.1.X?

We are planning to release various pre-release versions of the refactored app next week
